src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(int side, Color color) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public String getFigureInfo() {
+        return "Figure: Square, area: " + getArea() + " sq. units, side: "
+                + side + " units, color: " + getColor();
+    }
+    /* Wzór na pole kwadratu a*a */

```

Comment:
The comment on line 21 is in Polish. Consider translating it to English for consistency and to ensure that all developers can understand it.